algorithms/ar-bsrh/python2/binary_search.py

```diff
@@ -0,0 +1,27 @@
+def binarysearch(arr,x):
+  low=0
+  high=len(arr)-1
+  mid=0
+  while(low<=high): 
+    mid=(low+high)//2
+    if(arr[mid]<x):
+      low=mid+1
+    elif(arr[mid]>x):
+      high=mid-1
+    else: 
+      return mid
+  return -1
+  
+def main():
+  arr = [1,2,3,4,5,6]
+  index = binarysearch(arr,5)
+  if(index==-1):
+    print "The element is not found!"
```

Comment:
Codacy found an issue: [invalid syntax (F999)](https://app.codacy.com/gh/FOSS-UCSC/FOSSALGO/pullRequest?prid=6277716)

Reply:
you should use single quotations (not double quotations) with the print command. it is just a guidance rule.